jobs/config.json

```diff
@@ -6611,7 +6611,7 @@
       "--gcp-node-image=gci",
       "--gcp-zone=us-central1-f",
       "--provider=gke",
-      "--test_args=--ginkgo.focus=\\[sig-cli\\].*\\[Serial\\]|\\[Disruptive\\] --ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
+      "--test_args=--ginkgo.focus=\\[sig-cli\\].*\\[Serial\\]|\\[sig-cli\\].*\\[Disruptive\\] --ginkgo.skip=\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8",
```

Comment:
Because it is a logical "or".  Currently it is "[sig-cli] [Serial]" or "[Disruptive]" and it should be
"[sig-cli] [Serial]" "or" "[sig-cli] [Disruptive]".

You can see the problem [here](https://k8s-testgrid.appspot.com/sig-cli#gke-serial) where it shows non-sig-cli Disruptive tests on the dashboard.